tests/packages/cmake_defines/CMakeLists.txt

```
@@ -0,0 +1,20 @@
+cmake_minimum_required(VERSION 3.26)
```

```suggestion
cmake_minimum_required(VERSION 3.15)
```
Should add a badge for the minimum CMake

---

Done, thanks!